.jsdoc.js

```diff
@@ -23,23 +23,19 @@ module.exports = {
     template: './node_modules/ink-docstrap/template',
     recurse: true,
     verbose: true,
-    destination: './docs/'
+    destination: './docs/',
   },
-  plugins: [
-    'plugins/markdown'
-  ],
+  plugins: ['plugins/markdown'],
   source: {
     excludePattern: '(^|\\/|\\\\)[._]',
-    include: [
```

Comment:
the change in formatting here, is this the result of running the prettier?

Reply:
Yep yep.